### DISCUSSION TITLE
Built a Small E-Ink Tracker for Podcast Data

### DISCUSSION THREAD

**Discussion by SpencerWF** [skymethod/op3#27] [Show and tell]
Using a Raspberry Pi Zero, and E-Ink display and the OP3 API I built a small tracker that updates me every day of the downloads, and a comparison between the last 3 weeks of downloads.

![Sparkle Print](https://github.com/skymethod/op3/assets/6613586/469399b9-0234-4eb8-b833-cfa55e168804)

I used 6 triangles to represent the last 3 weeks of downloads.  The triangles are paired up where the coloured(Red or Yellow) triangle represents the total downloads of a given week, and the white triangle represents the downloads for the latest episode.  The left most coloured and white triangle represents the 3 weeks ago, the middle triangles are 2 weeks ago and are used as a reference for the other triangles, they shouldn't change in size, the right most triangles represent the last 7 days.  If a new episode was released this week then it will be the only week with a white triangle.

![IMG-20231118-WA0001](https://github.com/skymethod/op3/assets/6613586/190f4d68-2195-4b84-894b-00a1c857ff59)

Thanks for reading!

[Github](https://github.com/SpencerWF/podcast_metric_display)

[Link](https://spencerwf.com/#trackerdisplay)

**Comment by Dwev** [comment]
Looks great!

What does the $ value represent?  Is this V4V or ad revenue or something else?

**Reply by SpencerWF** [reply]
I also track a small Crypto investment on the same screen, not really relevant for OP3, but basically done the same way.  Could definitely be used to track ad revenue or similar.

**Comment by johnspurlock-skymethod** [comment]
So cool!

Do you have it mounted on a wall or desk somewhere?

**Reply by SpencerWF** [reply]
Hey, thanks so much! 

Currently it just sits on a table for me, but I'm thinking about re-writing the code with an embedded device (for lower battery draw), hooking it up to Lipo battery and mounting it somewhere.
